config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+root "instructors#index"
```

Comment:
We're missing an indentation here. Also, as an application convention, lets stick to single-quotes `'` unless we're using string interpolation (e.g: `"#{ruby-code-here}"`)